github/pulls_test.go

```diff
@@ -349,7 +350,8 @@ func TestPullRequestsService_Merge(t *testing.T) {
 			}`)
 	})
 
-	merge, _, err := client.PullRequests.Merge("o", "r", 1, "merging pull request")
+	options := &PullRequestOptions{Squash: false}
```

Comment:
nit: how about making this option `true` for the test... otherwise the option is unnecessary.